templates/desktop/developers.html

```diff
@@ -78,15 +78,23 @@
 
 <section class="row strip-light">
     <div class="strip-inner-wrapper">
-        <div class="eight-col append-four whats-new-graphic">
+        <div class="eight-col append-four">
             <h2>What&rsquo;s new in Ubuntu {{latest_release_full}}?</h2>
             <p class="eight-col">
-                Ubuntu {{latest_release_full}} brings the latest version of the Unity 8 developer preview which offers a basic suite of apps, including a browser and a terminal. To access it, just select a &lsquo;Unity 8&rsquo; session on the login screen.</p>
+                Ubuntu {{latest_release_full}}  brings a richer and simplified user experience with easier printing,  a redesigned productivity application and a number of application updates.</p>
+        </div>
+        <div class="twelve-col">
             <div class="combined-list">
+                <ul class="list-ticks six-col">
+                  <li>LibreOffice 5.3 with new UI for improved usability</li>
+                  <li>Fast install snaps with snap URL Support added to Ubuntu Software 3.22</li>
+                  <li class="last-item">Print without printer-specific drivers on selected printers</li>
+                </ul>
                 <ul class="list-ticks six-col last-col">
-                    <li>Unity 8 desktop interface (Ubuntu developer preview)</li>
-                    <li>Selected Unity 8 apps included: Ubuntu Browser, System Settings and Terminal</li>
-                    <li class="last-item">Install and run snaps from the Terminal</li>
+                  <li>Print without printer-specific drivers on selected printers</li>
+                  <li>Most GNOME applications updated to 3.24</li>
+                  <li>Linux 4.10 kernel</li>
+                  <li>New installations use swap file instead of swap partition</li>
```

Comment:
The last-item class is only required on the last item of the first list when in a combined-list container, so on small it adds a border, however, this seems to be broken site-wide. I’ll add a separate issue.